third_party/stablehlo/workspace.bzl

```diff
@@ -4,15 +4,15 @@ load("//third_party:repo.bzl", "tf_http_archive", "tf_mirror_urls")
 
 def repo():
     # LINT.IfChange
-    STABLEHLO_COMMIT = "371678eeb3995cf21b13c15947774ffaf02ab84a"
-    STABLEHLO_SHA256 = "373099bd70ee4340ed424a56182eae36f30bebfd7595fe3b849d1cb6f587ccb0"
+    STABLEHLO_COMMIT = "3800a6bf472644380cfb601eafaba88063bdec91"
+    STABLEHLO_SHA256 = "1eeffaa1e5f8a2044d738a8865a9327c153d21033efc045c8e1e23a21536cd6a"
     # LINT.ThenChange(Google-internal path)
 
     tf_http_archive(
         name = "stablehlo",
         sha256 = STABLEHLO_SHA256,
         strip_prefix = "stablehlo-{commit}".format(commit = STABLEHLO_COMMIT),
-        urls = tf_mirror_urls("https://github.com/openxla/stablehlo/archive/{commit}.zip".format(commit = STABLEHLO_COMMIT)),
+        urls = tf_mirror_urls("https://github.com/apivovarov/stablehlo/archive/{commit}.zip".format(commit = STABLEHLO_COMMIT)),
```

Comment:
Probably not intended to fetch it from your repo.

Reply:
If it's intended that you also update the deps in the same PR, could you split it in separate PRs?

Reply:
This PR is pending the merge of StableHLO https://github.com/openxla/stablehlo/pull/2482 Add f8E4M3 and f8E3M4 types support (in Review).

Reply:
Hi Alexander, Integrate StableHLO at openxla/stablehlo@4f31b2e7 was werged to XLA main today. It includes `float8_e4m3` type support. My temporary change in `third_party/stablehlo/workspace.bzl` was removed from this PR. @mooskagh